components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -63,6 +63,31 @@ const BillingSubscriptions: React.FC<Props> = () => {
 
     const [showCreditCardBanner, setShowCreditCardBanner] = useState(true);
 
+    const product = useSelector((state: GlobalState) => {
+        if (state.entities.cloud.products && subscription) {
+            const product = state.entities.cloud.products[subscription?.product_id];
+            return product.product_family === 'cloud' ? product : null;
```

Comment:
I believe the endpoint already returns all products that are only cloud products so this check here may not be required. @nickmisasi can correct me if I'm wrong here

Reply:
ok, @nickmisasi is that validation unnecessary?

Reply:
Endpoint does indeed only return cloud products so this is probably not needed.

Reply:
ok